cmake/mkldnn_omp.diff

```diff
@@ -0,0 +1,28 @@
+diff --git a/cmake/OpenMP.cmake b/cmake/OpenMP.cmake
```

Comment:
This file should be named mkldnn_omp.patch

Reply:
Combined the two patches and renamed the file to mkldnn.patch